cmd/podman/pull.go

```diff
@@ -50,14 +54,10 @@ func init() {
 	flags.StringVar(&pullCommand.CertDir, "cert-dir", "", "`Pathname` of a directory containing TLS certificates and keys")
 	flags.StringVar(&pullCommand.Creds, "creds", "", "`Credentials` (USERNAME:PASSWORD) to use for authenticating to a registry")
 	flags.BoolVarP(&pullCommand.Quiet, "quiet", "q", false, "Suppress output information when pulling images")
-
-	// Disabled flags for the remote client
-	if !remote {
-		flags.StringVar(&pullCommand.Authfile, "authfile", "", "Path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json. Use REGISTRY_AUTH_FILE environment variable to override")
-		flags.StringVar(&pullCommand.SignaturePolicy, "signature-policy", "", "`Pathname` of signature policy file (not usually used)")
-		flags.BoolVar(&pullCommand.TlsVerify, "tls-verify", true, "Require HTTPS and verify certificates when contacting registries")
-	}
-
+	flags.BoolVar(&pullCommand.TlsVerify, "tls-verify", true, "Require HTTPS and verify certificates when contacting registries")
```

Comment:
tls-verify was supposed to be "off" for the remote client.  have you changed your mind on it? if so, can you update the google doc that we used to review this so @jwhonce and i can be consistent?

Reply:
I don't remember why this was decided?  I am fine with turning it off, but I still think we need to be consistent between rootless and remote, (Except for when it makes little sense.)  I can turn this back off for remote, to get this PR in.  And then we can discuss at these `security` options, for non privileged users.